p4-16/psa/psa.p4

```diff
@@ -638,7 +640,11 @@ package PSA_Switch<IH, IM, EH, EM> (IngressPipeline<IH, IM> ingress,
 // constructor calls for PacketReplicationEngine and
 // BufferingQueueingEngine.
 
-#define PSA_SWITCH(ip, ep) PSA_Switch((ip), PacketReplicationEngine(), (ep), BufferingQueueingEngine())
+#define PSA_SWITCH(ip, ep) PSA_Switch(                           \
+                                      (ip),                      \
+                                      PacketReplicationEngine(), \
+                                      (ep),                      \
+                                      BufferingQueueingEngine())
 // END:Programmable_blocks
```

Comment:
We could consider moving this line '// END:Programmable_blocks' up to just before the macro is defined, if we are not sure we want to include it in the PSA spec itself, e.g. if the goal is to remove it.